src/server/ua_server_internal.h

```diff
@@ -74,8 +74,6 @@ struct UA_Server {
 #endif
 };
 
-void UA_Server_processBinaryMessage(UA_Server *server, UA_Connection *connection, UA_ByteString *msg);
```

Comment:
@joerg555 
Can we please leave this function in server_internal?
We don't want to expose it to the end user.

Making the bytestring const is good, though.